code/modules/heavy_vehicle/components/_components.dm

```diff
@@ -121,7 +121,7 @@
 		to_chat(user, "<span class='warning'>Turn \the [WT] on, first.</span>")
 		return
 	if(WT.remove_fuel(0, user))
-		var/repair_value = 15
+		var/repair_value = 30
```

Comment:
Not in favor of the (undocumented) repair speed increase here.

If something is able to take more damage, it should also take longer to repair it.